examples/cross_decomposition/plot_pcr_vs_pls.py

```diff
@@ -0,0 +1,110 @@
+"""
+==================================================================
+Principal Component Regression vs Partial Least Squares Regression
+==================================================================
+
+This example compares Principal Component Regression (PCR) and Partial Least
+Squares Regression (PLS) on a toy dataset. Our goal is to illustrate how PLS
+can outperform PCR when the target is strongly correlated with some features
+that have a low variance.
+
+PCR simply consists in applying :class:`~sklearn.decomposition.PCA` to the
+training data (possibly performing dimensionality reduction), and then
+training a regressor on the transformed training samples. In
+:class:`~sklearn.decomposition.PCA`, the transformation is purely
+unsupervized, meaning that no information about the targets is used. As a
+result, PCR may perform poorly in some datasets where the target is
+correlated with features that have a low variance. Indeed, the dimensionality
+reduction of PCA tries to only keep the features that have a high variance.
+Those that have a low variance will be dropped, and the final regressor will
```

Comment:
I'm not a fan of the use of "keep" and "drop" because strictly speaking that's not what's happening in PCA, but I can't put it simply

Suggestions welcome

Reply:
'use' and 'ignore'?